packages/design-system/src/components/DateField/DateField.stories.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import DateField from './DateField';
+import DateInput from './DateInput';
+
+export default {
+  title: 'Components/DateField',
+  component: DateField,
+  args: {
+    errorMessaage: 'Please enter a year in the past',
```

Comment:
I was just tying this out, and there's both an `errorMessaage` (typo), and an `errorMessage` control field. I've tried clearing out both, but the error message is still showing up in the component.

Reply:
Ah good catch, thats resolved, if you want to clear it out you have to enter an empty object like ""

Regarding the extra Story, it causes problems to have more than one of these in a story file so I'd need to create another file for it.. On the story page you can open the modal and change props while it's open, just not on the docs page which isn't really meant as a playground.  I think it's ok as is?

Reply:
Actually another story wont resolve anything but removing the button to open, it doesn't add any existing functionality or usefulness.

Reply:
Oh the issue with the errormessage was fixable by setting an argtype, my bad, fixed..

Reply:
> On the story page you can open the modal and change props while it's open,

Oh, you're right. This is enough then 👍